hs-bindgen/src/HsBindgen/C/Fold/Type.hs

```diff
@@ -579,10 +592,11 @@ partitionFields = go id where
 mkStructField ::
      ExtBindings
   -> CXTranslationUnit
-  -> DeclPath
+  -> Maybe CName         -- ^ Name of the struct (unless anonymous)
```

Comment:
This and the next field should really be (fieldName -> DeclPathCtxt); then this change would make sense.

Reply:
Addressed in https://github.com/well-typed/hs-bindgen/pull/523 .